README.md

```diff
@@ -41,9 +41,16 @@ Then, run the following commands in the root of the cloned repository:
     npm install
     npm install nodemon -g
     node force_update.js
+
+If you are on Windows, you will need to make sure you have `bash` installed somewhere and run the following command [with your `bash` path in place of the path below].
+
+    npm config set script-shell "C:\\Program Files\\git\\bin\\bash.exe"
+
+Then you can start the program like so:
+
     npm start
```

Comment:
I prefer the change in #554, as it is requires less steps for new contributors to setup the application for the first time. Maybe we can add both, and label this as an alternative for more advanced users.

Reply:
OK

Reply:
If you merge that one first I can deconflict

Reply:
Although I should also note that there are now three processes that run on npm start so it's going to be pretty annoying for windows people without this. I can put both in, though.